src/assets/style.scss

```diff
@@ -69,6 +69,14 @@ a {
   }
 }
 
+html[lang="es"] {
+  a:not([href^="http"]):not([href^="#"]):not([href*="/es/"]):not([href="/es"])[href*="/"]:after {
+    content: "(inglés)";
+    vertical-align: super;
+    font-size: 50%;
+  }
+}
```

Comment:
I know this selector is pretty crazy but I couldn't find a cleaner way to do it. It selects any hyperlink from a Spanish page to a non-Spanish page that isn't external, just a `#anchor` link, or just a local image link rather than a page path. It adds a little (ingles) marker to them so Spanish speakers know they're about to leave the Spanish content.